tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -44,7 +49,12 @@ export const validation = (fieldName, fieldValue) => {
  */
 function EditProfile() {
   const errorRef = useRef(null)
-  const stts = useSelector((state) => state.stts.stts)
+  const sttList = useSelector((state) => state.stts.sttList)
+  const requestedAccess = useSelector(
+    (state) => state.requestAccess.requestAccess
+  )
+  const sttAssigned = useSelector((state) => state.auth.user.stt)
```

Comment:
If both of these are true then we know that the user has already requested access. I need this for the `Redirect` below.